### ISSUE TITLE
No way to tell at a glance which builders have failed

### ISSUE THREAD

**Issue by dankegel** [buildbot/buildbot#5056]
Buildbot 0.8 had one crucial feature: builders whose most recent build had failed
were shown in red.

Buildbot 1.0 and up lack that feature.

Users here consider that a crtitical problem.  They feel bereft.  There's simply no way to
look at the buildbot and tell how healthy things are without clicking and clicking and clicking.

**Comment by tardyp** [comment]
I think waterfall has this feature

**Comment by dankegel** [comment]
No, because 1.x's waterfall only shows things built in the last XXX seconds.
My users visit the 1.x waterfall and come away wondering what has successfully built.  

And if you fixed the above problem, there's another one that makes the waterfall much less useful:
I don't know how to set up sub-waterfalls with the builders of interest for various teams as I could with 0.8's waterfall.

As a result of those two problems, 1.x's waterfall is completely worthless as a way of knowing what's broken.

**Comment by mojca** [comment]
It doesn't really help me either. The problems with the waterfall are the following:
* If a build is pending, this is no longer shown with 2.x.
* If a builder (worker) is offline, this is no longer visible.
* Builders which didn't build for a while vanish from the waterfall (I didn't find a super reliable way to show them all).
* I cannot make old (no longer existing) builders disappear from the waterfall until sufficient newer builds end up there.

**Comment by mojca** [comment]
@dankegel: Yes, tags are supposed to help and have recently been implemented by @rajdeepbharati, but I'm still experiencing some issues, the most important one probably being that the builds aren't reloaded based on currently visible builders, but are probably still just filtered in javascript, so one can easily end up with zero builds on the waterfall.

**Comment by survient** [comment]
Is there a way to do this via a custom build step? Are you able to add and remove failure tags to the builder as conditional steps?
